src/rgw/rgw_bucket.cc

```diff
@@ -560,14 +567,7 @@ int rgw_remove_bucket(RGWRados *store, rgw_bucket& bucket, bool delete_children)
 
   } while (!objs.empty());
 
-  string prefix, delimiter;
-
-  ret = abort_bucket_multiparts(store, cct, info, prefix, delimiter);
-  if (ret < 0) {
-    return ret;
-  }
-
-  ret = rgw_bucket_sync_user_stats(store, bucket.tenant, info);
+  ret = rgw_bucket_sync_user_stats(store, bucket.tenant, bucket.name);
```

Comment:
okay, i see that `bucket.tenant` was getting converted to `rgw_user` for the `(user_id, bucket_info)` overload. but the `(tenant_name, bucket_name)` overload has to call `get_bucket_info()` again to look up the `info` that we already have here. it looks like we can take a shortcut:
```c++
  ret = rgw_bucket_sync_user_stats(store, info.owner, info);
````